### PR TITLE
New Procedure Class

### PR DESCRIPTION
**Pull request by hguthrie**
This PR introduces a new Procedure class for DevDocs. The class is designed specifically as a title for introducing a set of steps (ordered list).

Because the Cloud guide used procedure titles with H4 headings extensively, this PR also applies the new procedure class to all of Cloud guide. All of the changes to Cloud guide were reviewed and merged into this branch, with the exception of the release note archive topic and some missed items. This provides a fix for one of the problems referenced in the Issue #5240 

Merged into this branch:

- #5443 
- #5444 
- #5457 
- #5474 
- #5476 

I checked the Cloud guide changes against the MD001 rule for heading order. I also scrubbed for H4 level usage and statements beginning with `"To "` and `"**To "`. This needs a final quick review to merge to master.

### CODE REVIEW

File: guides/v2.2/cloud/reference/cli-ref-topic.md [hunk 1]
```diff
@@ -12,67 +12,44 @@ You can install the Magento Cloud CLI when setting up your local environment for
...
-Some actions, such as adding a user, do not result in deployment.
-
-#### To create a new branch:
-
-```bash
-magento-cloud environment:branch <NAME> <PARENT_BRANCH>
-```
+Action | Command
+------ | --------
+To log in to a project: | `magento-cloud login`
```
Comment (meker12) [inline guides/v2.2/cloud/reference/cli-ref-topic.md]:
```suggestion
Log in to a project | `magento-cloud login`
```

File: guides/v2.2/cloud/reference/cli-ref-topic.md [hunk 2]
```diff
@@ -12,67 +12,44 @@ You can install the Magento Cloud CLI when setting up your local environment for
...
+To clone a project to a directory: | `magento-cloud project:get <PROJECT_ID> <DIRECTORY> -e <ENVIRONMENT_ID>`<br>**Note**: If you want to clone the `master` environment, omit the `-e <ENVIRONMENT_ID>` parameter.
```
Comment (meker12) [inline guides/v2.2/cloud/reference/cli-ref-topic.md]:
```suggestion
Clone a project to a directory | `magento-cloud project:get <PROJECT_ID> <DIRECTORY> -e <ENVIRONMENT_ID>`<br>**Note**: If you want to clone the `master` environment, omit the `-e <ENVIRONMENT_ID>` parameter.
```

**Comment by hguthrie** [comment]
@dshevtsov The class was already reviewed by Eugene. This is for you to see.

**Comment by hguthrie** [comment]
running tests

**Comment by hguthrie** [comment]
sorry for the error. Another case of "ohhh, darn 2.3 version of the topic!" grrr

**Comment by hguthrie** [comment]
running tests

**Pull request merged**